src/store/normalize-data.js

```diff
@@ -1,22 +1,48 @@
 import { getInitialPipelineState } from '../store/initial-state';
+import { arrayToObject } from '../utils';
+
 /**
  * Check whether data is in expected format
  * @param {Object} data - The parsed data input
  * @return {Boolean} True if valid for formatting
  */
 const validateInput = data => {
-  const { isArray } = Array;
-  return (
-    data && isArray(data.edges) && isArray(data.nodes) && isArray(data.tags)
-  );
+  if (!data) {
+    // Data may still be loading, so return initial state
```

Comment:
Might be clearer to say e.g 'so return false` rather than describing some external dependent result.